testsuite/features/support/commonlib.rb

```diff
@@ -601,14 +601,16 @@ def channel_sync_completed?(channel_name)
     elsif line.include?('Channel: ') && !line.include?(channel_name)
       channel_found = false
     elsif line.include?('Sync of channel completed.') && channel_found
-      return true
+      return true if channel_is_synced?(channel_name)
+
+      log "WARN: Repository metadata for #{channel_name} seems not synchronized. Even if the reposync log says it is."
```

Comment:
Text like that looks better to me:

```suggestion
      log "WARN: Repository metadata for #{channel_name} does not seem to be synchronized. Completion information in the log was not found."
```

Reply:
> Completion information in the log was not found.

No, that's not true, the information in the log is there, the problem is that it shouldn't be there :)